doc/components.md

```diff
@@ -23,47 +29,43 @@ The most commonly used implementation, `SpriteComponent`, can be created with a
     player.render(canvas); // it will render only if the image is loaded and the position and size parameters are not null
 ```
 
-In the event that you want to easily change the direction of your components rendering, you can also use
-`renderFlipX` and `renderFlipY` to flip anything drawn to canvas during `render(Canvas canvas)`.
-This is available on all `PositionComponent` objects, and is especially useful on `SpriteComponent` and
-`AnimationComponent`. Simply set `component.renderFlipX = true` for example reverse the horizontal rendering.
+## Component
+All components inherit from the abstract class `Component`.
```

Comment:
why changing the order? I feel we should talk about component before anything else.

Reply:
I think most of our users rarely interact directly with `Component` and very often with or close to `PositionComponent`, but I can change the order back if people think it is makes more sense to have in the hierarchical order.

Reply:
I think it makes sense to start describing component, but you can mention on the first paragraph that normally you use Position , with a link

Reply:
Sounds like a good compromise